### ISSUE TITLE
Unable to perform manual host announcement: cannot create new address beyond address gap limit

### ISSUE THREAD

**Issue by e-corp-sam-sepiol** [HyperspaceApp/Hyperspace#36]
This error is occurring on one of my Hyperspace hosts, and I believe it is keeping my host from being visible on the "active hosts" list @ spacehub.siahub.info with the inability to create new addresses.

I'm pretty sure this issue was created by either 0.2.0 or 0.2.2. 

```
pi@raspberrypi:~/go/bin $ ./hsc host announce ecorphosting.ddns.net:5583
Could not announce host: unable to perform manual host announcement: cannot create new address beyond address gap limit
```
```
pi@raspberrypi:~/go/bin $ ./hsc version
Hyperspace Client
        Version 0.2.3
        Git Revision db78391fd
        Build Time   Mon 19 Nov 22:31:01 UTC 2018
Hyperspace Daemon
        Version 0.2.3
        Git Revision db78391fd
        Build Time   Mon 19 Nov 22:31:01 UTC 2018
```

**Comment by afdy** [comment]
I had the same issue the other day on 0.2.3 however mine was with establishing contracts:

renter/contractor.log:2018/11/19 13:25:31.615922 contractmaintenance.go:884: Attempted to form a contract with <ip address>:5582, but negotiation failed: [cannot create new address beyond address gap limit]

Starting up hsd with the --address-gap-limit option resolved it.

I think this is a workaround rather than a fix, though. :) - not sure what the real problem is.

**Comment by e-corp-sam-sepiol** [comment]
I followed @afdy suggestion and increased the gap limit. It worked and I was able to manually announce my host. Console output provided below.

```
pi@raspberrypi:~/go/bin $ ./hsd --address-gap-limit 500 --host-addr :5583 &
[1] 14380
pi@raspberrypi:~/go/bin $ Hyperspace Daemon v0.2.3
Git Revision db78391fd
Loading...
(0/6) Loading hsd...
(1/6) Loading gateway...
(2/6) Loading consensus...
(3/6) Loading transaction pool...
(4/6) Loading wallet...
(5/6) Loading host...
(6/6) Loading renter...
Hyperspace Wallet Password found, attempting to auto-unlock wallet
Auto-unlock successful.
Finished loading in 2.85329984 seconds

pi@raspberrypi:~/go/bin $ ./hsc host announce ecorphosting.ddns.net:5583
Host announcement submitted to network.
The host has also been configured to accept contracts.
To revert this, run:
        hsc host config acceptingcontracts false

```

**Comment by huetsch** [comment]
I believe this issue has been resolved in v0.2.3. If you continue to see it beyond v0.2.3, please open a new ticket.
